docs: Explain how to work with auto-generated keys; Restructure to clarify how to specify keys for authoritative fields. (#205)
y-scope/yscope-log-viewer: docs/src/user-guide/struct-logs/index.md

# Overview

> The following are guides for working with structured logs.

Reviewer: ```suggestion
The guides below describe how to configure the log viewer to work with structured logs.
```

::::{grid} 1 1 2 2
:gutter: 2

:::{grid-item-card}
:link: format/index
Formatting as plain text
^^^

> A guide to formatting logs as plain text with a format string.

Reviewer: ```suggestion
How to format structured logs as plain text, using a format string.
```

"A guide to" feels redundant. I know this breaks the convention we have in the other docs, but I'd prefer to start making the improvement here.

:::

:::{grid-item-card}
:link: specifying-keys
Specifying keys
^^^

> A guide on how to specify keys in YScope syntax.

Reviewer: ```suggestion
How to specify keys when configuring the log viewer.
```
